src/main/java/com/xebialabs/deployit/ci/Credential.java

```diff
@@ -281,6 +281,7 @@ public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Project context) {
         }
 
         private FormValidation validateOptionalUrl(String url) {
+            Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);
```

Comment:
This is not expected as URL, so seems unnecessary. Better safe than sorry, I guess?

Reply:
I would keep it if ok.